config/routes.rb

```diff
@@ -12,6 +12,8 @@
   post '/carts', to: 'carts#create', as: 'carts'
   get '/cart', to: 'carts#showcart', as: 'showcart'
   post '/cart', to: 'carts#checkout_cart', as: 'checkout_cart'
+  delete '/cart', to: 'carts#delete_item', as: 'cart'
+  put '/cart', to: 'carts#update_item_quantity', as: 'update_item_quantity'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.